sct-commons/src/main/java/org/lfenergy/compas/sct/commons/ControlBlockService.java

```diff
@@ -1,159 +0,0 @@
-                                                                 NetworkRanges networkRanges, ControlBlockEnum controlBlockEnum) {
-        PrimitiveIterator.OfLong appIdIterator = Utils.sequence(networkRanges.appIdStart(), networkRanges.appIdEnd());
-        Iterator<String> macAddressIterator = Utils.macAddressSequence(networkRanges.macAddressStart(), networkRanges.macAddressEnd());
-
-        SclRootAdapter sclRootAdapter = new SclRootAdapter(scd);
-        return sclRootAdapter.streamIEDAdapters()
-                .flatMap(iedAdapter ->
-                        iedAdapter.streamLDeviceAdapters()
-                                .filter(LDeviceAdapter::hasLN0)
-                                .map(LDeviceAdapter::getLN0Adapter)
-                                .flatMap(ln0Adapter -> ln0Adapter.streamControlBlocks(controlBlockEnum))
-                                .map(controlBlockAdapter -> configureControlBlockNetwork(controlBlockNetworkSettings, appIdIterator, macAddressIterator, controlBlockAdapter)))
-                .flatMap(Optional::stream)
-                .toList();
-    }
-
-    private Optional<SclReportItem> configureControlBlockNetwork(ControlBlockNetworkSettings controlBlockNetworkSettings, PrimitiveIterator.OfLong appIdIterator, Iterator<String> macAddressIterator, ControlBlockAdapter controlBlockAdapter) {
-        SettingsOrError settingsOrError = controlBlockNetworkSettings.getNetworkSettings(controlBlockAdapter);
-        if (settingsOrError.errorMessage() != null) {
-            return Optional.of(controlBlockAdapter.buildFatalReportItem(
-                    "Cannot configure network for this ControlBlock because: " + settingsOrError.errorMessage()));
-        }
-        Settings settings = settingsOrError.settings();
-        if (settings == null) {
-            return Optional.of(controlBlockAdapter.buildFatalReportItem(
-                    "Cannot configure network for this ControlBlock because no settings was provided"));
-        }
-        if (settings.vlanId() == null) {
-            return Optional.of(controlBlockAdapter.buildFatalReportItem(
-                    "Cannot configure network for this ControlBlock because no Vlan Id was provided in the settings"));
-        }
-        if (!appIdIterator.hasNext()) {
-            return Optional.of(controlBlockAdapter.buildFatalReportItem(
-                    "Cannot configure network for this ControlBlock because range of appId is exhausted"));
-        }
-        if (!macAddressIterator.hasNext()) {
-            return Optional.of(controlBlockAdapter.buildFatalReportItem(
-                    "Cannot configure network for this ControlBlock because range of MAC Address is exhausted"));
-        }
-
-        return controlBlockAdapter.configureNetwork(appIdIterator.nextLong(), macAddressIterator.next(), settings.vlanId(), settings.vlanPriority(),
```

Comment:
This method is no longer used but still exists.

https://github.com/com-pas/compas-sct/blob/feat/362_RSR-812_Use_generated_JAXB_class_for_configuration_of_Communication_of_Control_Blocks/sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ied/ControlBlockAdapter.java#L143C34-L143C34

Reply:
You're right. I remove it.